02-documentation/developer-guides/building-a-nextjs-blog-with-squidex.md

```diff
@@ -0,0 +1,588 @@
+
+Open your terminal and run the following command to install Next.js and its dependencies.
+
+```bash
+npx create-next-app squidex-blog
+```
+
+If you are prompted to confirm that you want to install the create-next-app package, type `y` and press enter to continue with the installation.
+
+Once the Next.js is set up, run the following commands to enter the `squidex-blog` directory and start the development server.
+
+```bash
+cd squidex-blog
+yarn dev
+```
+
+The `yarn dev` command starts a Next.js development server at `http://localhost:3000`. Visit `htttp://localhost:3000` to see your Next.js app. You would see _Welcome to Next.js_ on the page. You have now successfully installed Next.js and are ready to start building your blog.
+
+## Step 4: Create global styles and a Layout component
+
+You would start by creating a `Layout` component that adds a header and footer to all the pages of your blog. Create a `components` folder in `squidex-blog` and create a `layout.js` file in the `squidex-blog/components` folder with the following contents:
+
+```jsx
+import Link from "next/link";
+
+export default function Layout({ children }) {
+  return (
+    <div className="container">
+      <div className="header">
+        <Link href="/">
+          <a className="blog-heading">My Blog</a>
+        </Link>
+      </div>
+      {children}
+      <div className="footer">
+        <div>My Blog</div>
+        <div>(c) 2022 </div>
+      </div>
+    </div>
+  );
+}
+```
+
+This creates a footer and a header with a link to your blog's homepage. The `children` prop allows you to wrap whatever content is added into the `Layout` component in the header and footer.
+
+After creating the `Layout` component, replace the contents of the `squidex-blog/styles/globals.css` file with the following:
+
+```css
+html,
+body {
+  padding: 0;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu,
+    Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 1em;
+}
+
+.header {
+  margin-top: 2em;
+  margin-bottom: 4em;
+}
+
+.blog-heading {
+  font-size: 2em;
+  font-weight: bold;
+  text-decoration: none;
+  color: black;
+}
+
+.footer {
+  margin: 4em auto;
+  width: fit-content;
+}
+
+.blog-item {
+  margin-top: 20px;
+}
+
+.blog-item-title {
+  font-size: 2em;
+  text-decoration: none;
+  color: black;
+}
+
+.blog-title {
+  font-size: 3em;
+}
+```
+
+Now that you have modified the website's stylesheet, you will go on to modify the custom Next.js `App` component of your blog to wrap the application in a header and footer with the `Layout` component you created. In Next.js, a custom app component is used to add a persistent layout to app pages and to add global CSS to your app. You can learn more about it from the [Custom App page of the Next.js documentation](https://nextjs.org/docs/advanced-features/custom-app).
```

Comment:
The first sentence is very long and I have no idea what you mean with the second sentence.  What is a persistent layout? Do you mean consistent?